node/resolvers/Mutations/Users.ts

```diff
@@ -412,7 +412,12 @@ const Users = {
         id,
         userId,
       })
-      .then((result: any) => {
+      .then(async (result: any) => {
+        await events.sendEvent('', 'b2b-organizations-graphql.removeUser', {
```

Comment:
I'm not very familiar with colossus events, but in general we usually don't wait for events like this to successfully finish before returning the response of an API. We don't want to response to the removeUser API to fail just because we failed to send the event, right? 

So I believe that we should remove the `await` here and let the event be silently sent on the background, returning the response regardless of its success, since the event doesn't alter the actual operation's success.

So what I'm suggesting here is:

```suggestion
       // Send an event on the background, but don't wait to respond.
        events.sendEvent('', 'b2b-organizations-graphql.removeUser', {
```

Reply:
That makes total sense @mairatma . I've just removed it